pkg/apis/core/v1alpha1/storagecluster.go

```diff
@@ -139,6 +185,9 @@ type StorageSpec struct {
 	// UseAllWithPartitions use all available unformatted devices
 	// including partitions. This will be ignored if Devices is not empty.
 	UseAllWithPartitions *bool `json:"useAllWithPartitions"`
+	// Force use the drives even if there is file system present on it. Note
+	// that the drives will be wiped before using.
+	Force *bool `json:"force"`
```

Comment:
Can we make this more descriptive like `ForceUseDisks`? `Force` could mean any number of things